pallets/ajuna-awesome-avatars/src/types/avatar/versions/v2/mod.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(unused_imports)]
```

Comment:
Yeah, the thing is, if I remove it and the fix the issues I get compilation errors, so this error is a false flag right now I believe.

Reply:
Did you have to fix the issues because you have bumped the rust-tool chain? Usually, I prefer if a version bump is done in a separate PR to not introduce more unrelated changes in the PR.